pkgs/build-support/docker/examples.nix

```diff
@@ -105,7 +105,8 @@ rec {
   };
 
   # 6. nix example to play with the container nix store
-  # docker run -it --rm nix nix-store -qR $(nix-build '<nixpkgs>' -A nix)
+  # docker load < $(nix-build '<nixpkgs>' -A dockerTools.examples.nix)
+  # docker run -it --rm nix /bin/bash
```

Comment:
This no longer check the database is well initialized. In fact, to run this command, the `dockerTools.buildImage` function is sufficient.